webmention.py

```diff
@@ -67,6 +68,14 @@ def discover(url, **requests_kwargs):
       logger.debug(f'Webmention discovery: got endpoint in tag: {endpoint}')
       return Endpoint(endpoint, resp)
 
+  # If we are not currently following a client-side redirect https://www.w3.org/TR/WCAG20-TECHS/H76.html
+  if not follow_meta_refresh:
+    http_equiv = util.parse_http_equiv(soup)
+    if http_equiv: # else implicit break out and continue like normal
+      endpoint = util.fragmentless(urljoin(url, http_equiv))
+      logger.debug(f'Webmention discovery: got http_equiv in tag: {endpoint}')
+      return discover(endpoint, follow_meta_refresh=True)
```

Comment:
Consider checking that `url != endpoint`? It wouldn't catch multi-hop redirect loops, but that's ok.

Reply:
Also pls pass `**requests_kwargs` too

Reply:
should be fixed. let me know if this needs any changing